fast64_internal/oot/scene/exporter/to_c/scene_table_c.py

```diff
@@ -103,12 +103,13 @@ def getSceneParams(scene, exportInfo, sceneNames):
     # you just have to make it return something other than None, not necessarily a string
     sceneIndex = getSceneIndex(sceneNames, bpy.context.scene.ootSceneExportSettings.option)
     sceneName = sceneTitle = sceneID = sceneUnk10 = sceneUnk12 = None
+    name = scene.name if scene is not None else exportInfo.name
 
     # if the index is None then this is a custom scene
     if sceneIndex is None and scene is not None:
         sceneName = scene.name.lower() + "_scene"
         sceneTitle = "none"
-        sceneID = "SCENE_" + (scene.name.upper() if scene is not None else exportInfo.name.upper())
+        sceneID = ootSceneNameToID.get(name, f"SCENE_{name.upper()}")
```

Comment:
we need the new IDs and the old names, new IDs to compile and old names for the files (since the folders still use debug names), this part got weird with the newest changes (and I wasn't really inspired tbh)

Reply:
yeeaaah I guess this makes assets really weird to handle

For example what happens if I add a custom scene named "ydan", it's partially going to conflict with deku tree?
Well whatever I hope there won't be too many bugs